portafly/src/i18n/setupTranslate.tsx

```diff
@@ -0,0 +1,8 @@
+import Polyglot from 'node-polyglot'
+
+export const setupTranslate = (polyglot:Polyglot , locale: string, phrases: object) => {
+  const t = polyglot.t.bind(polyglot)
+  polyglot.locale(locale)  
+  polyglot.extend({ ...phrases })
```

Comment:
I think this is unnecessary:
```suggestion
  polyglot.extend(phrases)
```